insights/core/__init__.py

```diff
@@ -765,9 +765,29 @@ class YAMLParser(Parser, LegacyItemAccess):
     A parser class that reads YAML files.  Base your own parser on this.
     """
 
+    ignore_lines = []
+    """
+    The first keywords of the lines that need to ignored in the parsing process.
+    * All keywords added in this list must be in lowercase.
+    """
+
     def parse_content(self, content):
+        def _ignore_lines(content, ignore_lines):
+            idxs = []
+            if content and ignore_lines:
+                for idx, line in enumerate(content):
+                    # Skipping lines start with any keyword of ignore_lines
+                    if line.strip().lower().startswith(tuple(ignore_lines)):
```

Comment:
What about making this `tuple(ignore_lines)` conversion ahead of this for loop? so that it can be reused inside the loop.

Reply:
Nice catching.